tests/test_prometheus_connect.py

```diff
@@ -52,6 +52,7 @@ def test_get_metric_range_data(self):  # noqa D102
             < metric_objects_list[0].end_time.timestamp(),
             "invalid metric end time",
         )
+        self.assertFalse(end_time < start_time, "invalid metric end time")
```

Comment:
Can you please write a test case where you use the invalid start_time and end_time in the function that you updated?
Something like this: https://github.com/AICoE/prometheus-api-client-python/blob/559434db91bd59af7adadc41456a3f889f09ddb6/tests/test_prometheus_connect.py#L91-L96